core/src/main/java/io/micronaut/core/graal/AutomaticFeatureUtils.java

```diff
@@ -44,67 +45,118 @@
     /**
      * Marks the given class to be initialized at build time, only if it is present.
      *
-     * @param access the {@link BeforeAnalysisAccess} instance
-     * @param className the class name
+     * @param access     The {@link BeforeAnalysisAccess} instance
+     * @param classNames The class names
      */
-    public static void initializeAtBuildTime(BeforeAnalysisAccess access, String className) {
-        findClass(access, className).ifPresent(RuntimeClassInitialization::initializeAtBuildTime);
+    public static void initializeAtBuildTime(BeforeAnalysisAccess access, String... classNames) {
```

Comment:
Even if the class is flagged as internal. Should not we keep the method and annotate it with `@Deprecated` and in the java doc tell to use the new one: 

```java
    public static void initializeAtBuildTime(BeforeAnalysisAccess access, String className) {
```

From the old one we can the new one.

Reply:
No, we don't. I asked about this at one engineering meeting and it's totally ok to do "whatever" we want with `@Internal` classes. Besides that, both signatures are compatible.